### PR TITLE
Issue #13213: Remove '//ok' comments  from input files

### PR DESCRIPTION
**Pull request by aye-nyeinSan**
#13213

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -68,6 +68,28 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
+  <suppress id="UnnecessaryOkComment"
```
Comment (romani) [inline config/checkstyle-input-suppressions.xml:71]: We need to remove such suppressions.
But you are adding them.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (aye-nyeinSan) [reply config/checkstyle-input-suppressions.xml:71]: Now I got it!! I am working on it @romani .

Comment (aye-nyeinSan) [reply config/checkstyle-input-suppressions.xml]: @romani, I have changed files and made PR again. Could you please kindly check it?

**Comment by romani** [comment]
Please make single commit in PR and look at CI failures and address them.

**Pull request closed without merging**